Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcess.py

```diff
@@ -158,19 +158,27 @@ def validateInputs(self):
         return result
 
     def setUp(self):
-        self.sample = self.getPropertyValue('SampleRuns').split(',')
-        self.absorber = self.getPropertyValue('AbsorberRuns').split(',')
-        self.beam = self.getPropertyValue('BeamRuns').split(',')
-        self.flux = self.getPropertyValue('FluxRuns').split(',')
-        self.container = self.getPropertyValue('ContainerRuns').split(',')
+        self.sample = self.getPropertyValue('SampleRuns') \
```

Comment:
I'm not sure you need this for whatever is a MultipleFileProperty; it should take care of sanitizing the input and return the formatted strings of fully resolved file path lists. I think you need to do this only for the ones that are just string properties, so the sensitivity, masks and references.

Reply:
Ah yes, you are right. I thought that the spaces would stay in the strings but the MultipleFileProperty is taking care of that.

I am reverting it for the MultipleFileProperty.